.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+services:
+  - docker
+env:
+  - DOCKER_COMPOSE_VERSION=1.22.0
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - docker-compose build
+  - docker-compose run --rm app /bin/sh -c "bundle install; RAILS_ENV=test  bundle exec rails db:create db:migrate"
```

Comment:
Why do you need `bundle install` here? The image should already have gems installed.

Reply:
I checked, gems not installed